neqo-transport/src/cc/cubic.rs

```diff
@@ -13,19 +15,38 @@ use neqo_common::qtrace;
 
 use crate::cc::classic_cc::WindowAdjustment;
 
-// CUBIC congestion control
-
-// C is a constant fixed to determine the aggressiveness of window
-// increase  in high BDP networks.
+/// > C is a constant fixed to determine the aggressiveness of window
+/// > increase  in high BDP networks.
+///
+/// <https://www.rfc-editor.org/rfc/rfc8312.html#section-4.1>
```

Comment:
I kinda prefer linking to the datatracker HTMLized version, but YMMV.

Reply:
:+1: changed in https://github.com/mozilla/neqo/pull/2344/commits/528338ba543ab5125f7a3e86fcffaaf4e46b2dad.